src/app/components/arlas-wui-root/arlas-wui-root.component.ts

```diff
@@ -1124,50 +1129,74 @@ export class ArlasWuiRootComponent implements OnInit, AfterViewInit, OnDestroy {
       if (!!resultListContributor) {
         const idFieldName = this.collectionToDescription.get(resultListContributor.collection).id_path;
         const id = feature.properties[idFieldName.replace(/\./g, '_')];
-        resultListContributor.detailedDataRetriever.getData(id).subscribe(
-          data => {
-            const rowItem = new Item([], new Map());
-            rowItem.identifier = id;
-            resultListContributor.detailedDataRetriever.getActions(rowItem).subscribe(ac => rowItem.actions = ac);
-            const detail = new Array<{
-              group: string;
-              details: Array<{
-                key: string;
-                value: string;
-              }>;
-            }>();
-            data.details.forEach((k, v) => {
-              const details = new Array<{
-                key: string;
-                value: string;
-              }>();
-              k.forEach((value, key) => {
-                details.push(
-                  { key, value }
-                );
-              });
-              detail.push({
-                group: v,
-                details: details
-              });
-            });
-            rowItem.itemDetailedData = detail;
-            const popupContent = this.dynamicComponentService.injectComponent(
-              ResultDetailedItemComponent,
-              x => {
-                x.rowItem = rowItem;
-                x.actionOnItemEvent = this.actionOnPopup;
-                x.idFieldName = idFieldName;
-              });
-            if (!!this.popup) {
-              this.popup.remove();
+        // Open the list panel if it's closed
+        this.disableRecalculateExtend = true;
+        if (!this.resultlistService.listOpen) {
+          this.toggleList();
+        }
+        // Select the good tab if we have several
+        // No tabs case
+        if (this.resultlistContributors.length === 1) {
+          this.waitFor(this.resultListComponent,() => this.openDetail(id));
+        } else {
+          this.waitFor(this.resultListComponent,() => {
+            // retrieve list
+            const tab = document.querySelector('[aria-label="' + resultListContributor.identifier + '"]') as any;
+            tab.click();
+            // Set Timeout to wait the new tab
+            setTimeout(() => this.openDetail(id), 250);
+            this.disableRecalculateExtend = false;
```

Comment:
this disabling could happen early as the `openDetail` is executed after and `openDetail` it self has timeouts inside, no ?